Test/spec/simd/meta/simd_i64x2_arith.py

```diff
@@ -36,14 +36,15 @@ def hex_unary_op_test_data(self):
 
     @property
     def i64x2_i8x16_test_data(self):
+        """This test data will be intepreted by the SIMD.v128_const() method in simd.py."""
         return {
             'i64x2.add': [
-                [['0x7fffffffffffffff', ['0', '0', '0', '0', '0', '0', '0', '0x80'] * 4], '-1',
+                [['0x7fffffffffffffff', ['0', '0', '0', '0', '0', '0', '0', '0x80'] * 2], '-1',
                  ['i64x2', 'i8x16', 'i64x2']],
                 [['1', '255'], '0', ['i64x2', 'i8x16', 'i64x2']]
             ],
             'i64x2.sub': [
-                [['0x7fffffffffffffff', ['0', '0', '0', '0', '0', '0', '0', '0x80'] * 4], '-1',
+                [['0x7fffffffffffffff', ['0', '0', '0', '0', '0', '0', '0', '0x80'] * 2], '-1',
```

Comment:
Do these changes (4 -> 2) affect the produced `.wast` files?

Reply:
No, it doesn't.